`make depends target=aarch64-linux-gnu` fails

Hi there

I'm trying to cross compile `v0.17.2.3` for `aarch-linux-gnu` on `Pop!_OS 21.04`

This is the error when trying to build:
```
$ git clone --recursive https://github.com/monero-project/monero.git -b v0.17.2.3
$ make -j9 depends target=aarch64-linux-gnu
[...]
cd contrib/depends && make HOST=aarch64-linux-gnu && cd ../.. && mkdir -p build/aarch64-linux-gnu/release
make[1]: Entering directory '/home/robbie/Documents/monero/contrib/depends'
Building unwind...
make[2]: Entering directory '/home/robbie/Documents/monero/contrib/depends/work/build/aarch64-linux-gnu/unwind/1.2-89d5f33f1be'
Making all in src
make[3]: Entering directory '/home/robbie/Documents/monero/contrib/depends/work/build/aarch64-linux-gnu/unwind/1.2-89d5f33f1be/src'
make  all-am
make[4]: Entering directory '/home/robbie/Documents/monero/contrib/depends/work/build/aarch64-linux-gnu/unwind/1.2-89d5f33f1be/src'
make[4]: Nothing to be done for 'all-am'.
make[4]: Leaving directory '/home/robbie/Documents/monero/contrib/depends/work/build/aarch64-linux-gnu/unwind/1.2-89d5f33f1be/src'
make[3]: Leaving directory '/home/robbie/Documents/monero/contrib/depends/work/build/aarch64-linux-gnu/unwind/1.2-89d5f33f1be/src'
Making all in tests
make[3]: Entering directory '/home/robbie/Documents/monero/contrib/depends/work/build/aarch64-linux-gnu/unwind/1.2-89d5f33f1be/tests'
/bin/bash ../libtool --tag=CC   --mode=link aarch64-linux-gnu-gcc  -pipe  -fexceptions -Wall -Wsign-compare  -L/home/robbie/Documents/monero/contrib/depends/aarch64-linux-gnu/lib -o Lperf-simple Lperf-simple.o ../src/libunwind.la
libtool: link: aarch64-linux-gnu-gcc -pipe -fexceptions -Wall -Wsign-compare -o Lperf-simple Lperf-simple.o  -L/home/robbie/Documents/monero/contrib/depends/aarch64-linux-gnu/lib ../src/.libs/libunwind.a -lc -lgcc_s
/usr/lib/gcc-cross/aarch64-linux-gnu/10/../../../../aarch64-linux-gnu/bin/ld: ../src/.libs/libunwind.a(dyn-info-list.o):(.bss+0x0): multiple definition of `_U_dyn_info_list'; ../src/.libs/libunwind.a(Linit.o):(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:752: Lperf-simple] Error 1
make[3]: Leaving directory '/home/robbie/Documents/monero/contrib/depends/work/build/aarch64-linux-gnu/unwind/1.2-89d5f33f1be/tests'
make[2]: *** [Makefile:480: all-recursive] Error 1
make[2]: Leaving directory '/home/robbie/Documents/monero/contrib/depends/work/build/aarch64-linux-gnu/unwind/1.2-89d5f33f1be'
make[1]: *** [funcs.mk:261: /home/robbie/Documents/monero/contrib/depends/work/build/aarch64-linux-gnu/unwind/1.2-89d5f33f1be/./.stamp_built] Error 2
make[1]: Leaving directory '/home/robbie/Documents/monero/contrib/depends'
make: *** [Makefile:50: depends] Error 2
```

My system specs are as follows:
* Intel i7-6700HQ (4 core, 8 thread)
* 32GiB DDR3 Memory (not sure frequency, doubt it matters)

I installed the build dependencies like so:
```sh
$ sudo apt-get update
$ sudo apt-get dist-upgrade -y
$ sudo apt-get install -y build-essential \
    cmake pkg-config libboost-all-dev \
    libssl-dev libzmq3-dev libunbound-dev \
    libsodium-dev libunwind8-dev liblzma-dev \
    libreadline6-dev libldns-dev libexpat1-dev \
    doxygen graphviz libpgm-dev qttools5-dev-tools \
    libhidapi-dev libusb-1.0-0-dev libprotobuf-dev \
    protobuf-compiler libudev-dev git

$ sudo apt-get install -y libgtest-dev && \
  cd /usr/src/gtest && \
  sudo cmake . && \
  sudo make && \
  sudo mv lib/libg* /usr/lib/

$ sudo apt-get install -y \
    g++-arm-linux-gnueabihf g++-aarch64-linux-gnu \
    gperf
```
And python3 is my default set python
```sh
$ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1
```

Maybe I'm just missing something stupid.

--- a/contrib/depends/packages/unwind.mk
+++ b/contrib/depends/packages/unwind.mk
@@ -1,8 +1,8 @@
 package=unwind
-$(package)_version=1.2
+$(package)_version=1.5.0
 $(package)_download_path=https://download.savannah.nongnu.org/releases/libunwind
 $(package)_file_name=lib$(package)-$($(package)_version).tar.gz
-$(package)_sha256_hash=1de38ffbdc88bd694d10081865871cd2bfbb02ad8ef9e1606aee18d65532b992
+$(package)_sha256_hash=90337653d92d4a13de590781371c604f9031cdb50520366aa1e3a91e1efb1017
 $(package)_patches=fix_obj_order.patch
 
 define $(package)_preprocess_cmds
@@ -12,7 +12,7 @@ endef
 define $(package)_config_cmds
   cp -f $(BASEDIR)/config.guess config/config.guess &&\
   cp -f $(BASEDIR)/config.sub config/config.sub &&\
-  $($(package)_autoconf) --disable-shared --enable-static AR_FLAGS=$($(package)_arflags)
+  $($(package)_autoconf) --disable-shared --enable-static --disable-tests --disable-documentation AR_FLAGS=$($(package)_arflags)
 endef
 
 define $(package)_build_cmds

--- a/contrib/depends/patches/unwind/fix_obj_order.patch
+++ b/contrib/depends/patches/unwind/fix_obj_order.patch
@@ -1,9 +1,9 @@
---- config/ltmain.sh.O	2017-01-13 16:00:54.000000000 +0000
-+++ config/ltmain.sh	2019-11-17 06:46:51.994402494 +0000
-@@ -7957,6 +7957,8 @@
- 	    esac
- 	  done
+--- config/ltmain.sh.0	2020-11-10 17:25:26.000000000 +0100
++++ config/ltmain.sh	2021-09-11 19:39:36.000000000 +0200
+@@ -10768,6 +10768,8 @@
  	fi
+ 	func_to_tool_file "$oldlib" func_convert_file_msys_to_w32
+ 	tool_oldlib=$func_to_tool_file_result
 +	oldobjs=`for obj in $oldobjs; do echo $obj; done | sort`
 +	oldobjs=" `echo $oldobjs`"
  	eval cmds=\"$old_archive_cmds\"

